### PR TITLE
[IOTDB-2844] Implementaion of AggregationOperator and RawDataAggregationOperator

### PR DESCRIPTION
**Pull request by Alima777**
The design doc is in https://y8dp9fjm8f.feishu.cn/docs/doccnuuXYJo0AeM34YhCq61o6Ce#

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/execution/schedule/DriverScheduler.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public static DriverScheduler getInstance() {
 
   private static final int MAX_CAPACITY = 1000; // TODO: load from config files
   private static final int WORKER_THREAD_NUM = 4; // TODO: load from config files
-  private static final int QUERY_TIMEOUT_MS = 10_000; // TODO: load from config files or requests
+  private static final int QUERY_TIMEOUT_MS = 1000_000; // TODO: load from config files or requests
```
Comment (liuminghui233) [inline server/src/main/java/org/apache/iotdb/db/mpp/execution/schedule/DriverScheduler.java:69]: Is this a temporary value for debug ?

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/LocalExecutionPlanner.java [hunk 2]
```diff
@@ -530,7 +536,64 @@ public Operator visitOffset(OffsetNode node, LocalExecutionPlanContext context)
...
+            inputLocationList.add(new InputLocation[] {inputLocationParts.get(0).get(i)});
+          } else {
+            inputLocationList.add(
+                new InputLocation[] {
+                  inputLocationParts.get(0).get(i), inputLocationParts.get(1).get(i)
+                });
+          }
+        }
+
+        // TODO type here may be incorrect
```
Comment (liuminghui233) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/LocalExecutionPlanner.java:572]: why?

File: server/src/main/java/org/apache/iotdb/db/query/expression/Expression.java [hunk 3]
```diff
@@ -70,7 +70,7 @@
 public abstract class Expression {
 
   /////////////////////////////////////////////////////////////////////////////////////////////////
-  // Expression type inferring for execution plan generation
+  // Expression type inferring for execution plan generation //////////////////////////////////////
```
Comment (liuminghui233) [inline server/src/main/java/org/apache/iotdb/db/query/expression/Expression.java:73]: useless change

**Review by liuminghui233 — APPROVED** [review]

File: server/src/main/java/org/apache/iotdb/db/mpp/execution/schedule/DriverScheduler.java (hunk 1 above)
Comment (Alima777) [reply server/src/main/java/org/apache/iotdb/db/mpp/execution/schedule/DriverScheduler.java]: It's changed to one minute now.

**Pull request merged**